chart/values.yaml

```diff
@@ -414,3 +414,20 @@ loki-stack:
               - __meta_kubernetes_pod_uid
               - __meta_kubernetes_pod_container_name
               target_label: __path__
+
+call-home:
```

Comment:
I need to check if hyphens are considered okay now in template key-strings, iirc this was not considered a good practice and helm has a function to allow templating the values where the keys with hyphens exist.

Reply:
RIght! Good catch.

Reply:
hyphens are not usable.